packages/zeplin-css-extension/src/generator.js

```diff
@@ -41,14 +41,26 @@ class CSS {
         });
     }
 
+    declaration(d) {
+        return `${INDENTATION}${d.name}${MIDFIX} ${d.getValue(this.params, this.variables)}${SUFFIX}`;
+    }
+
+    declarationsBlock(declarations) {
+        return `{\n${declarations.map(this.declaration, this).join("\n")}\n}`;
+    }
+
     variable(name, value) {
         return `${PREFIX}${name}${MIDFIX} ${value.toStyleValue(this.params)}${SUFFIX}`;
     }
 
     ruleSet({ selector, declarations }, { parentDeclarations = [] } = {}) {
         const filteredDeclarations = this.filterDeclarations(declarations, parentDeclarations);
-        return `${selector} {\n${filteredDeclarations.map(p => `${INDENTATION}${p.name}${MIDFIX} ${p.getValue(this.params, this.variables)}${SUFFIX}`).join("\n")}\n}`;
+        return `${selector} ${this.declarationsBlock(filteredDeclarations)}`;
+    }
+
+    atRule({ identifier, declarations }) {
+        return `@${identifier} ${this.declarationsBlock(declarations)}`;
```

Comment:
We need to filter declarations in here too. This way, it does not respect `SHOW_DEFAULT_VALUES` option.

Reply:
For font-face definitions, I think we should keep the default values; they are somewhat different from applying rules to an element, and they define what that specific font corresponds to. I guess it works fine with default values like you said, but I feel like the redundancy here makes it more understandable.